index.js

```diff
@@ -252,7 +262,7 @@ HtmlWebpackPlugin.prototype.executeTemplate = function (templateFunction, chunks
     .then(function () {
       var templateParams = {
         compilation: compilation,
-        webpack: compilation.getStats().toJson(),
+        webpack: getStats(compilation),
```

Comment:
From the doc, this should be `compilation.getStats()`, right?

> The following variables are available in the template:
>    * webpack: the webpack stats object.
(source https://github.com/jantimon/html-webpack-plugin#writing-your-own-templates)